dotCMS/src/integration-test/java/com/dotcms/util/AnnotationUtilsTest.java

```diff
@@ -0,0 +1,69 @@
+package com.dotcms.util;
+
+import com.dotcms.util.annotationtest.TestActionlet1;
+import com.dotcms.util.annotationtest.TestActionlet2;
+import com.dotcms.util.annotationtest.TestJsonSerializer;
+import com.dotcms.util.annotationtest.TestResource;
+import com.dotmarketing.portlets.workflows.actionlet.Actionlet;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import javax.ws.rs.Path;
+import java.util.Set;
+
+public class AnnotationUtilsTest {
+
+    @BeforeClass
+    public static void prepare() throws Exception {
```

Comment:
Codacy found an issue: [A method/constructor should not explicitly throw java.lang.Exception](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=4888160)